client/light/src/blockchain.rs

```diff
@@ -129,10 +129,10 @@ impl<S, Block> BlockchainBackend<Block> for Blockchain<S> where Block: BlockT, S
 		Err(ClientError::NotAvailableOnLightClient)
 	}
 
-	fn extrinsic(
+	fn transaction(
```

Comment:
Renamed this to `transaction` since it now returns an indexed portion, rather than all of the extrinsic data